api/grpcserver/grpcserver_test.go

```diff
@@ -228,8 +229,26 @@ func TestMain(m *testing.M) {
 }
 
 func newNIPostWithChallenge(challenge *types.Hash32, poetRef []byte) *types.NIPost {
+	tree, err := merkle.NewTreeBuilder().
+		WithLeavesToProve(map[uint64]bool{0: true}).
+		Build()
+	if err != nil {
+		panic("failed to add leaf to tree")
+	}
```

Comment:
```suggestion
	require.NoError(t, err, "failed to add leaf to tree")
```

Reply:
You mean the `nipost` global variable? Can we make that variable local instead? It is only used in 2 places (both in the same test) 🙂